retrofit-converters/protobuf/src/main/resources/META-INF/proguard/retrofit2-proguard-converter.pro

```diff
@@ -0,0 +1,7 @@
+# Parser looked up reflectively.
```

Comment:
Should this file be named `retrofit2-protobuf-converter.pro`?

Reply:
Yep want to send a PR?

Reply:
#3623